yasnippet.el

```diff
@@ -3035,18 +3035,30 @@ through the field's start point"
    (not (and (yas--field-number field)
              (zerop (yas--field-number field))))))
 
-(defun yas--snippets-at-point (&optional all-snippets)
-  "Return a sorted list of snippets at point.
-
-The most recently-inserted snippets are returned first."
-  (sort
-   (delq nil (delete-dups
-              (mapcar (lambda (ov) (overlay-get ov 'yas--snippet))
-                      (if all-snippets (overlays-in (point-min) (point-max))
-                        (nconc (overlays-at (point))
-                               (overlays-at (1- (point))))))))
-   #'(lambda (s1 s2)
-       (<= (yas--snippet-id s2) (yas--snippet-id s1)))))
+(defun yas-active-snippets (&optional where)
+  "Return a sorted list of active snippets.
+The most recently-inserted snippets are returned first.
+
+By default, lists snippets around point.  If optional argument
```

Comment:
I decided this makes sense, and have gone with the `(&optional beg end)` + refer to `overlays-in` approach.